### PR TITLE
As per the GitHub webcast

### PR DESCRIPTION
**Pull request by marcad-owner**
Initiate from the owner's account.

### CODE REVIEW

**Comment by marcad-owner** [comment]
Explicitly asking @marcad-contributor to participate.

**Comment by marcad-contributor** [comment]
Righty-ho. Shall take a peek.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,9 @@
...
-collaboration exploration
+# Test Repository 
+
+This is a test repository to practice proper GIT methodology between an owner and a contributor who will fork the project.
+
+## Owner
+* @marcad-owner is the owner
+
+## Contributors
+* TBD
```
Comment (marcad-contributor) [inline README.md]: Go ahead and add me to the list of contributors

Comment (marcad-contributor) [inline README.md]: This is a line note. Hmmm.

Comment (marcad-owner) [inline README.md]: This is my line note (from marcad-owner)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,9 @@
-# test1
-collaboration exploration
+# Test Repository 
+
+This is a test repository to practice proper GIT methodology between an owner and a contributor who will fork the project.
```
Comment (marcad-contributor) [inline README.md:3]: This was added by clicking on line 3. In theory, I'd comment on the text of that paragraph.

**Pull request merged**